ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink

```diff
@@ -4,13 +4,21 @@
 set -eu
 # set -o pipefail # TODO: pipefail needs more rework in the script
 
+mesh_vpn_enabled="$(uci get wireguard.mesh_vpn.enabled)"
+
+if [[ "${mesh_vpn_enabled}" == "0" ]]; then
+	# Stop the script if mesh_vpn is disabled
+	exit 0
+fi
+
 if { set -C; true 2>/dev/null >/var/lock/checkuplink.lock; }; then
 	trap "rm -f /var/lock/checkuplink.lock" EXIT
 else
-	echo "Lock file exists... exiting"
+	logger -p err -t checkuplink "Lock file exists... exiting."
```

Comment:
This isn't really an error, it probably is better to be a notice. This is triggered when the code below takes longer than the rerun from the cronjob. Happens regularly on slow connections.

Reply:
sounds good to me